test/helper-config.ts

```diff
@@ -6,3 +6,7 @@ export const INVALID_EMAIL: string = process.env.INVALID_EMAIL ?? "invalid";
 export const LOGIN_URL: string =
   process.env.LOGIN_URL ?? "http://localhost:8080/account/login";
 export const SIGNUP_URL: string = process.env.SIGNUP_URL ?? " ";
+export const AUTHOR_IMAGE_URL: string = process.env.AUTHOR_IMAGE_URL ?? " ";
+export const DEFAULT_AUTHOR_IMAGE: string =
```

Comment:
you can place all those path and URLs into the AuthorImage page object.
And I don't think you need to read them from the env. variable. I cannot see a case where they would change and needed to be set by env. variable